extensions/pkg/controller/dnsrecord/reconciler.go

```diff
@@ -121,7 +118,7 @@ func (r *reconciler) reconcile(ctx context.Context, dns *extensionsv1alpha1.DNSR
 		return reconcile.Result{}, err
 	}
 
-	r.logger.Info("Starting the reconciliation of dnsrecord", "dnsrecord", dns.Name)
+	r.logger.Info("Starting the reconciliation of dnsrecord", "dnsrecord", kutil.ObjectName(dns))
```

Comment:
> Used `kutil.ObjectName` everywhere, used lowercase `dnsrecord` (as opposed to `DNSRecord`) in all messages, and also improved the phrasing of a few messages. If you like it, I could make these changes across all extension reconcilers in another PR (they are suffering from the same minor bugs / inconsistencies).

Yes, I like it very much and it definitely makes sense to make such cleanup/improvement in all (extension) controllers.

If you will do it, I would propose to construct resource-specific loggers first via `.WithValues()` and then pass it around, so one doesn't have to remember to add the resource name to every log statement.

Reply:
OK, I agree, will note it down and open a PR at some point.